kustomize/overlays/prod/shard/patches/deploy-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: event-bridge-shard-operator-config
+data:
+  EVENT_BRIDGE_MANAGER_URL: "http://event-bridge:8080"
+  EVENT_BRIDGE_INGRESS_IMAGE: "quay.io/5733d9e2be6485d52ffa08870cabdee0/ingress:latest"
```

Comment:
Same as before: we start with latest and the images will be updated with another PR when they will be ready